packages/edit-post/src/components/visual-editor/index.native.js

```diff
@@ -93,7 +63,7 @@ class VisualEditor extends Component {
 					isFullyBordered={ isFullyBordered }
 					rootViewHeight={ rootViewHeight }
 					safeAreaBottomInset={ safeAreaBottomInset }
-					isPostTitleSelected={ this.state.isPostTitleSelected }
+					isPostTitleSelected={ this.props.isPostTitleSelected }
```

Comment:
Just curious, is there a reason to use `isPostTitleSelected` selector here in visual-editor and pass the resulting value down to the `BlockList` as a prop instead of just using the `isPostTitleSelected` selector directly inside of `BlockList` itself?

Reply:
I simply wanted to minimize the changes to the native part here. https://github.com/WordPress/gutenberg/pull/16677 will be responsible of refactoring the VisualEditor and moving the block picker out of BlockList into an `Inserter` component.